client/src/templates/Challenges/codeally/show.tsx

```diff
@@ -304,14 +304,12 @@ class ShowCodeAlly extends Component<ShowCodeAllyProps> {
                       <Spacer size='medium' />
                     </>
                   )}
-                <Alert id='codeally-cookie-warning' variant='info'>
-                  <p>{t(`intro:misc-text.enable-cookies`)}</p>
-                </Alert>
                 <Button
                   aria-describedby='codeally-cookie-warning'
                   block={true}
                   variant='primary'
                   data-cy='start-codeally'
+                  disabled={true}
```

Comment:
This does not really matter much, but a client-side disable does not stop anything - someone can still invoke/access `tryToShowCodeAlly` with a disabled button.

That is to say, we might want to remove the `iframe` altogether? _I do not know why I keep reviewing drafts_ 🤪

Reply:
✔️

Reply:
This PR is meant to just quickly disable the browser version of RDB. I plan to clean up all the code with the PR that enables the gitpod setup.